tests/FieldDescription/FilterTypeGuesserTest.php

```diff
@@ -44,18 +47,21 @@ protected function setUp(): void
      * @dataProvider provideGuessCases
      *
      * @phpstan-param class-string $expectedType
+     * @phpstan-param array<string, mixed> $fieldMapping
      */
     public function testGuess(
         int|string|null $mappingType,
         string $expectedType,
         array $expectedOptions,
-        int $expectedConfidence
+        int $expectedConfidence,
+        array $fieldMapping = []
     ): void {
         $fieldDescription = $this->createStub(FieldDescriptionInterface::class);
         $fieldDescription->method('getFieldName')->willReturn('foo');
         $fieldDescription->method('getMappingType')->willReturn($mappingType);
         $fieldDescription->method('getParentAssociationMappings')->willReturn([]);
         $fieldDescription->method('getTargetModel')->willReturn('Foo');
+        $fieldDescription->method('getFieldMapping')->willReturn($fieldMapping);
```

Comment:
Would getFieldInfo or getInfo fits better? 

No blocker

Reply:
I'm pretty sure I'm not understanding your suggestion :smile: 
Here, the mocked method is [`FieldDescriptionInterface::getFieldMapping()`](https://github.com/sonata-project/SonataAdminBundle/blob/724c4fcec44b33c74aa3f3cc37627aa2e7890446/src/FieldDescription/FieldDescriptionInterface.php#L164). It is called at [`FilterTypeGuesser::guess()`](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1768/files#diff-eaa783586b4ba6a39f34afac400bf906a0fe4401f7e5d129d5435733c97bc941R73), but I'm not sure what should I rename :thinking:

Reply:
I was on the phone and I thought that getFieldMapping is a newly introduced method in this PR, nvm